components/fields/input.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormikProps } from 'formik';
+import { ChangeEvent } from 'react';
+
+type IInput = FormikProps<any> & {
```

Comment:
Keep some sensable names, e.g. `InputComp`. this applies to every other file.

Reply:
ok sure

Reply:
- progress bar is pending will be finish by tomorrow
- naming of file and variable changed
- env.json is the constant file I will change the file name to config.json
-  ok I will try to add is accordion way
- I will add testcases